helm_chart/values.yaml

```diff
@@ -0,0 +1,95 @@
+arch:
+  ppc64le: "3 - Most preferred"
+  amd64: "0 - Do not use"
+  s390x: "0 - Do not use"
+
+image:
+  pullPolicy: "Never"
+  restartPolicy: "Always"
+
+authService:
+  name: "auth-service"
+  image: "acmeair_auth_service_liberty1:latest"
+  replicas: 1
+  port: 80
+
+bookingService:
+  name: "booking-service"
+  image: "acmeair_booking_service_liberty1:latest" 
+  replicas: 1
+  port: 80
+
+customerService:
+  name: "customer-service"
+  image: "acmeair_customer_service_liberty1:latest"
+  replicas: 1
+  port: 80
+
+mainService:
+  name: "main-service"
+  image: "acmeair_main_service_liberty1:latest"
+  replicas: 1
+  port: 80
+
+flightService:
+  name: "flight-service"
+  image: "acmeair_flight_service_liberty1:latest"
+  replicas: 1
+  port: 80
+
+supportService:
+  name: "support-service"
+  image: "acmeair_support_service_liberty1:latest"
+  replicas: 1
+  port: 80
+
+custIngress:
+  # The host name must be DNS resolvable
+  host: "cluster-1518518865078-master.openpowercontainer.com"
```

Comment:
This is not needed and should be left empty.. Applies to all the below 'host' key

Reply:
I am able to address other comments (and updated the PR) except for this one. If I set `host` to empty then I see the problem of http getting redirected to https as well as all microservices APIs returning 301 status.